docs/types/conversion.rst

```diff
@@ -188,6 +188,11 @@ Addresses
 As described in :ref:`address_literals`, hex literals of the correct size that pass the checksum
 test are of ``address`` type. No other literals can be implicitly converted to the ``address`` type.
 
-Explicit conversions from ``bytes20`` or any integer type to ``address`` result in ``address payable``.
+Explicit conversions from ``bytes20`` or ``uint160`` to ``address`` is allowed and result in ``address payable``.
 
 An ``address a`` can be converted to ``address payable`` via ``payable(a)``.
+
+.. note::
+    Prior to version 0.8.0, it was possible to convert explicitly any integer type (signed or unsigned) of any size 
+    (from ``int8``/``uint8`` to ``int256``/``uint256``) to an address. This was removed in 0.8.0, and only conversion 
```

Comment:
`chk_coding_style` job is failing because of this:
```suggestion
    Prior to version 0.8.0, it was possible to convert explicitly any integer type (signed or unsigned) of any size
    (from ``int8``/``uint8`` to ``int256``/``uint256``) to an address. This was removed in 0.8.0, and only conversion
```
Please configure your editor to automatically remove trailing whitespace.